client/flutter/lib/l10n/intl_es.arb

```diff
@@ -1,23 +1,26 @@
 {
-    "worldHealthOrganization": "",
-    "learnMore": "",
-    "protectYourself": "",
-    "checkYourHealth": "",
-    "localMaps": "",
-    "shareTheApp": "",
-    "healthCheck": "",
-    "feelingDistressed": "",
-    "washHands": "",
-    "cougningAndSneezing": "",
-    "throwAwayTissue": "",
-    "washHandsFrequently": "",
-    "socialDistancing": "",
-    "seekMedicalCare": "",
-    "infoForEveryone":"",
-    "infoForParents":"",
-    "childStress":"",
-    "childLove":"",
-    "childDistance":"",
-    "childRoutines":"",
-    "childFacts":""
+    "worldHealthOrganization": "Organización Mundial de la Salud",
+    "learnMore": "Aprenda más",
+    "protectYourself": "Protéjase",
+    "checkYourHealth": "Consulte su estado de salud",
+    "localMaps": "Mapas Locales",
+    "shareTheApp": "Compartir la aplicación",
+    "healthCheck": "Consulta de salud",
+    "feelingDistressed": "Se siente angustiado/a?",
+    "washHands": "Lávese las manos a menudo con agua y jabón",
+    "cougningAndSneezing": "Cuando tosa o estornude, tápese la boca y la nariz con un pañuelo, o con su codo",
```

Comment:
```suggestion
    "coughingAndSneezing": "Cuando tosa o estornude, tápese la boca y la nariz con un pañuelo, o con su codo",
```

Reply:
Looks like that key needs to be changed everywhere.

Reply:
Yep... I think that should be done in a different pull request though